builder/virtualbox/ovf/builder.go

```diff
@@ -4,28 +4,141 @@ import (
 	"errors"
 	"fmt"
 	"log"
+	"strings"
 
 	"github.com/mitchellh/multistep"
 	vboxcommon "github.com/mitchellh/packer/builder/virtualbox/common"
 	"github.com/mitchellh/packer/common"
 	"github.com/mitchellh/packer/helper/communicator"
+	"github.com/mitchellh/packer/helper/config"
 	"github.com/mitchellh/packer/packer"
+	"github.com/mitchellh/packer/template/interpolate"
 )
 
 // Builder implements packer.Builder and builds the actual VirtualBox
 // images.
 type Builder struct {
-	config *Config
+	config Config
```

Comment:
I sort of did this blindly in that I don't know what it means. I only changed it, so it'd match: https://github.com/mitchellh/packer/blob/master/builder/vmware/iso/builder.go#L23

It could very easily have been the wrong thing to do.